flink-runtime/src/main/java/org/apache/flink/streaming/api/operators/StreamSource.java

```diff
@@ -52,7 +52,6 @@ public class StreamSource<OUT, SRC extends SourceFunction<OUT>>
     public StreamSource(SRC sourceFunction, boolean emitProgressiveWatermarks) {
         super(sourceFunction);
 
-        this.chainingStrategy = ChainingStrategy.HEAD;
```

Comment:
Does this mean that we have all chainingStrategy settings in OperatorFactory. So where is the HEAD set here?